src/EventStore.Core/TransactionLog/Chunks/TFChunk/TFChunk.cs

```diff
@@ -250,6 +245,13 @@ private enum CacheStatus {
 				throw new CorruptDatabaseException(new ChunkNotFoundException(_filename));
 
 			_fileSize = (int)fileInfo.Length;
+			_handle = File.OpenHandle(
+				_filename,
+				FileMode.Open,
+				FileAccess.Read,
+				FileShare.ReadWrite,
+				_reduceFileCachePressure ? FileOptions.None : FileOptions.RandomAccess | FileOptions.SequentialScan);
```

Comment:
some parenthesis would be handy to make the intention clear
is it weird to be setting both the randomaccess and sequentialscan flags? if its intentional it probably warrants a comment

Reply:
It is intentional, old code uses both options on different instances of `FileStream` (sequential for bulk reader and random for reader work item).

Reply:
Fixed